compiler/lib/config.ml

```diff
@@ -58,6 +58,8 @@ module Flag = struct
 
   let inline = o ~name:"inline" ~default:true
 
+  let effects = o ~name:"effects" ~default:true
```

Comment:
We should probably change the default for now.

Reply:
Indeed. I think this is convenient for testing, for now.

Reply:
I've pushed changes to explicitly enable effects for tests that need it.